go.mod

```diff
@@ -19,7 +19,6 @@ require (
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/stretchr/objx v0.5.0 // indirect
```

Comment:
is this the result of `go mod tidy`? Should we add it to the check list?

Reply:
I mean.. if we are touching all repos already it could make sense